_api-reference/ingest-apis/simulate-ingest.md

```diff
@@ -36,48 +34,54 @@ POST /_ingest/pipeline/35678/_simulate
 ```
 {% include copy-curl.html %}
 
-## Path and HTTP methods
+## Example requests
```

Comment:
This is not just a complete example, but PATH and HTTP method. You need a body for the request.

```
GET _ingest/pipeline/_simulate
POST _ingest/pipeline/_simulate
```